multiqc.wdl

```diff
@@ -2,8 +2,9 @@ version 1.0
 
 task MultiQC {
     input {
-        String? preCommand
-        File analysisDirectory
+        String dockerTag = "1.7--py_1"
+        # Use a string here so cromwell does not relocate an entire analysis directory
+        String analysisDirectory
```

Comment:
This will not work with the cloud then.

Reply:
Nope. We will look for a workaround when the time is there.